airflow_dbt_python/hooks/dbt.py

```diff
@@ -413,287 +378,3 @@ def setup_dbt_logging(self, debug: Optional[bool]):
             file_log.propagate = False
             configured_file.setLevel("INFO")
             configured_file.propagate = False
-
-    def get_dbt_target_from_connection(
```

Comment:
Ah, I just finished reviewing the previous commit, and now this is all gone. I think some of my comments may no longer be relevant.

Next time, it will be easier to review if we split the work into multiple PRs, or at least squash some commits.